lake-manifest.json

```diff
@@ -15,7 +15,7 @@
    "type": "git",
    "subDir": null,
    "scope": "",
-   "rev": "95eff173b853f2ce190e3c085d7bdcf349be71f6",
+   "rev": "c44e0c8ee63ca166450922a373c7409c5d26b00b",
    "name": "mathlib",
    "manifestFile": "lake-manifest.json",
    "inputRev": null,
```

Comment:
Shouldn't it be: 

```
"inputRev": "v4.19.0"
```

?

Reply:
Yeah just to reiterate this: when we do version bumps the diff should look something like https://github.com/ImperialCollegeLondon/FLT/pull/398 so we know we're compatible with the corresponding mathlib tag.

Reply:
Done.